package.json

```diff
@@ -26,6 +29,12 @@
     "chokidar": "^3.0.2"
   },
   "devDependencies": {
-    "standard": "^13.1.0"
+    "electron": "^8.1.1",
+    "rimraf": "^3.0.2",
```

Comment:
This has got to go: I don't want to have an extra dependency (even if only dev) to remove a single file.

Reply:
I think he used `rimraf` to remove the previously generated `.d.ts` file. This is only used for windows users. Mac and Linux users already have `rm rf` commands on their terminal.

Reply:
Got it working without rimraf